### PR TITLE
Milestone 2 - Structure

### PR DESCRIPTION
**Pull request by men32z**
The basic structure of the project wich includes
- folder structure
- Basic components, App, Book 
- Basic Containers, BooksForm, BookList
- Redux integration
- Reducers and Actions.

### CODE REVIEW

**Comment by men32z** [comment]
*** I have Microverse permit to work alone on this project.

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+    { id: getID(), title: 'Star Wars', category: 'History' },
+    { id: getID(), title: 'Scary Movie', category: 'Horror' },
+    { id: getID(), title: 'React Basics', category: 'Learning' },
+    { id: getID(), title: 'H.P.', category: 'Kids' },
+  ],
+};
+
+const store = createStore(rootReducer, defaultProps);
+
+ReactDOM.render(<Provider store={store}><App /></Provider>, document.getElementById('root'));
```
Comment (azdrenymeri) [inline src/index.js]: Based on the requirements of the project you should wrap `App` with `Provider` and assing the whole thing to a variable, then pass it to the `ReactDOM.render()` function.
ex:
```javascript
const wrappedApp = (
  <Provider store={store}>
    <App />
  </Provider>
);
ReactDOM.render(wrappedApp, document.getElementById('root'));
```

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @men32z :wave: 
### Good job on this milestone :+1: 
#### There is an error with prop types inside `BookList` Container.
scr

### Kindly fix this error and add it for another review

File: src/index.js (hunk 1 above)
Comment (men32z) [reply src/index.js]: Thanks I'll fix the errors.

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED 

Hi @men32z good job on making changes, everything works fine now, please merge and move to the next milestone :+1:  

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**